pkg/kubectl/cmd/run.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const (
 	run_long = `Create and run a particular image, possibly replicated.
-Creates a replication controller to manage the created container(s).`
+Creates a deployment or job to manage the created container(s).`
```

Comment:
hm, I see you have removed the rc and pod generators below. Isn't this going to break people?